### PR TITLE
Fix exception message

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Fix exception message

## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

### CODE REVIEW

File: ols/src/cache/cache.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def _check_user_id(user_id: str) -> None:
         """Check if given user ID is valid."""
         # TODO: needs to be updated when we know the format
         if Cache.COMPOUND_KEY_SEPARATOR in user_id:
-            raise ValueError("Incorrect user ID {user_id}")
+            raise ValueError(f"Incorrect user ID {user_id}")
```
Comment (bparees) [inline ols/src/cache/cache.py:27]:
```suggestion
            raise ValueError(f"Invalid user ID {user_id}")
```

File: tests/unit/cache/test_in_memory_cache.py [hunk 2]
```diff
@@ -53,9 +53,9 @@ def test_get_nonexistent_user(cache):
 
 def test_get_improper_user_id(cache):
     """Test how improper user ID is handled."""
-    with pytest.raises(ValueError, match="Incorrect user ID"):
+    with pytest.raises(ValueError, match="Incorrect user ID :"):
```
Comment (bparees) [inline tests/unit/cache/test_in_memory_cache.py:56]:
```suggestion
    with pytest.raises(ValueError, match="Invalid user ID :"):
```

File: tests/unit/cache/test_in_memory_cache.py [hunk 3]
```diff
@@ -53,9 +53,9 @@ def test_get_nonexistent_user(cache):
...
         cache.get(":", conversation_id)
-    with pytest.raises(ValueError, match="Incorrect user ID"):
+    with pytest.raises(ValueError, match="Incorrect user ID foo:bar"):
```
Comment (bparees) [inline tests/unit/cache/test_in_memory_cache.py:58]:
```suggestion
    with pytest.raises(ValueError, match="Invalid user ID foo:bar"):
```

File: tests/unit/cache/test_redis_cache.py [hunk 4]
```diff
@@ -45,9 +45,9 @@ def test_get_nonexistent_key(cache):
 
 def test_get_improper_user_id(cache):
     """Test how improper user ID is handled."""
-    with pytest.raises(ValueError, match="Incorrect user ID"):
+    with pytest.raises(ValueError, match="Incorrect user ID :"):
```
Comment (bparees) [inline tests/unit/cache/test_redis_cache.py:48]:
```suggestion
    with pytest.raises(ValueError, match="Invalid user ID :"):
```

File: tests/unit/cache/test_redis_cache.py [hunk 5]
```diff
@@ -45,9 +45,9 @@ def test_get_nonexistent_key(cache):
...
         cache.get(":", conversation_id)
-    with pytest.raises(ValueError, match="Incorrect user ID"):
+    with pytest.raises(ValueError, match="Incorrect user ID foo:bar"):
```
Comment (bparees) [inline tests/unit/cache/test_redis_cache.py:50]:
```suggestion
    with pytest.raises(ValueError, match="Invalid user ID foo:bar"):
```

**Review by bparees — COMMENTED** [review]
basically we should have 2 kinds of errors:

1) invalid - the formatting is not correct or the userid or conversationid is missing entirely
2) not found - we got a userid/conversationid header, but we couldn't find a matching conversation in the cache

for case (2) we should log it and also return some indicator in the Response api object so the UI can warn the user, but it should not fail the overall request.  It just means the generated response won't benefit from conversation history context.

File: tests/unit/cache/test_in_memory_cache.py (hunk 3 above)
Comment (tisnik) [reply tests/unit/cache/test_in_memory_cache.py]: fixed

File: tests/unit/cache/test_redis_cache.py (hunk 4 above)
Comment (tisnik) [reply tests/unit/cache/test_redis_cache.py]: fixed

File: tests/unit/cache/test_redis_cache.py (hunk 5 above)
Comment (tisnik) [reply tests/unit/cache/test_redis_cache.py]: fixed

Comment (tisnik) [reply tests/unit/cache/test_redis_cache.py]: fixed all @bparees

**Pull request merged**